sdk/storage/azure_storage_blob/src/clients/mod.rs

```diff
@@ -6,7 +6,9 @@
 mod blob_client;
 mod blob_container_client;
 mod blob_service_client;
+mod block_blob_client;
 
 pub use blob_client::BlobClient;
 pub use blob_container_client::BlobContainerClient;
 pub use blob_service_client::BlobServiceClient;
+pub use block_blob_client::BlockBlobClient;
```

Comment:
You need to export all the client options as well. See #2523. The `clients` module should always export all clients and their options (the ones you want exported at all, that is...obviously not the generated ones you're wrapping).

Reply:
Hi Heath, I did see that issue and was holding off on doing this until I got some in-flight PRs in (this would be the last one). The next PR will be regenerating against v14 to address the breaking changes, and to address your concerns raised above. Thanks!